data-prepper-plugins/ml-inference-processor/src/main/java/org/opensearch/dataprepper/plugins/ml_inference/processor/MLProcessorConfig.java

```diff
@@ -89,6 +92,10 @@ public class MLProcessorConfig {
     @JsonProperty("retry_time_window")
     private Duration retryTimeWindow = DEFAULT_RETRY_WINDOW;
 
+    @JsonPropertyDescription("The retry interval for the throttled records. Default is 60s.")
+    @JsonProperty(value = "retry_interval_seconds", defaultValue = "" + DEFAULT_RETRY_INTERVAL)
+    private int retry_interval_seconds =  DEFAULT_RETRY_INTERVAL;
```

Comment:
I would use `Duration` type for this and rename to `retry_interval`. With duration type users can either put an ISO_8601 time ("PT3M") or a simple duration time ("60s"). You can validate it with annotations (ex: https://github.com/opensearch-project/data-prepper/blob/fc0b563b46204d4195a083d5f235264897112f6f/data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/cloudwatch_logs/config/ThresholdConfig.java#L49)